src/api/config/routes.rb

```diff
@@ -769,6 +769,7 @@ def self.public_or_about_path?(request)
       resource :staged_requests, controller: 'staging/staged_requests', only: [:create, :destroy], constraints: cons
     end
 
+    resources :excluded_requests, controller: 'staging/excluded_requests', only: [:index], constraints: cons
```

Comment:
there is a conflict with https://github.com/openSUSE/open-build-service/pull/7161/commits/0dc1b55cf2866a6c39d7e2714d78d71dd4d811ff (just so that you take it into account if the other PR gets merge first)